content/admin/org_settings/integrations/Agile-central.md

```diff
@@ -0,0 +1,86 @@
+
+> **Notes:** 
+* To find your Agile Central API key, log in to the [Agile Central Application manager](https://help.rallydev.com/rally-application-manager), and go to the **API Keys** tab.  
+* Contrast saves the username, password and Agile Central URL entered in your configuration as a set of credentials. See the **Credential Sets** section for more details.
+
+* Once you complete the fields, click the button to **Test connection**. The test verifies that Contrast can reach the Agile Central instance and that the specified user can log in.
+
+<a href="assets/images/Agile-central-connect.png" rel="lightbox" title="Configure a new Agile Central integration"><img class="thumbnail" src="assets/images/Agile-central-connect.png"/></a>
+
+* Once connected, select the **Applications** that you want to be available to this integration. 
+* Choose a **Project Name** and **Owner** from the dropdown menus. 
+* In the **Default Priority** section, use the dropdown menus to choose a priority level for each vulnerability severity. 
+* Choose the **Environment** for which you want to generate tickets. 
+* Choose a **Defect State**. 
+* Add a name that the tickets are **Submitted By**. 
+
+<!-- 
+> **Note:** If you change the Project or Issue type, required and additional fields are updated. However, the UI keeps the selected values that apply to the new configuration.
+
+ -->
+
+>**Note:** While none of these configuration fields are required, Agile Central may populate tickets with their own default values for any fields you leave blank. 
+
+To add another integration once you're connected in Contrast, click the **Add Configuration** link in Agile Central row.
+
+### Automatically create tickets 
+
+To automatically create tickets for newly discovered vulnerabilities, check the designated box in the configuration form. In the multiselect field that appears, choose the Rule(s) and/or Severity level(s) of the vulnerabilities for which you want to generate tickets. 
+
+>**Note:** This selection doesn't generate tickets retroactively. 
+
+
+## Edit Configurations
+
+To see existing configurations, click the **Show Configurations** link in Agile Central row. Click on a configuration name to go to the **Agile Central Connection** form, and edit the field values. You can also delete your configuration by clicking the trashcan icon. 
+
+## Credentials
+
+Contrast saves the latest set of credentials that you enter in your Agile Central configurations to help you set up new connections even faster. The username, password and URL values that you enter in your first configuration become the default credentials for your following configurations. In subsequent configurations, Contrast will auto-populate the fields with the default credentials, but allow you to modify the values as needed. You can also manage your saved sets of credentials to simultaneously update all of the affected configurations. 
```

Comment:
Since this credential section is mentioned in this page (Agile Central) I think you should change _username_ and _password_ for **API Key**